src/hardware/vmware.cpp

```diff
@@ -21,9 +21,20 @@ CHECK_NARROWING();
 // Drivers:
 // - https://git.javispedro.com/cgit/vbados.git
 // - https://github.com/NattyNarwhal/vmwmouse (warning: release 0.1 is unstable)
+// - official Windows 9x VMware mouse driver
 
+static bool is_interface_enabled = false;
+static bool has_feature_mouse    = false;
+
+// Whether Intel 8042 entry point API is currently enabled
+bool is_i8042_unlocked = false;
```

Comment:
static?

Reply:
Definitely - I have missed this one. Changed.